<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_add</class>
 <widget class="QDialog" name="user_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>회원 등록</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>차량번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>전화번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>정기주차</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>포인트</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="plate_txt">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_txt">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="point_txt">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="apply_btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>신청</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="not_apply_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>신청안함</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(148, 198, 65);
	color: rgb(255, 255, 255);
	border-radius: 5px
}
QPushButton:hover{
	
	background-color: rgb(78, 154, 6);
	border-radius: 5px
}</string>
   </property>
   <property name="text">
    <string>수정</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
